### ISSUE TITLE
Option inIncludedFiles is too greedy

### ISSUE THREAD

**Issue by krisztianb** [krisztianb/typedoc-plugin-replace-text#5]
Using a plugin config like this doesn't produce the expected results:

```json
"replaceText": {
    "inCodeCommentText": false,
    "inCodeCommentTags": false,
    "inIncludedFiles": true,
    "replacements": [
        {
            "pattern": "A",
            "replace": "B"
        }
    ]
}
```

**Expected**

Once would expect that `A` is replaced by `B` only in the `README` and other included markdown files as suggested by the plugin's README.

**Actual**

With the config above `A` is replaced everywhere by `B` even in code comment text and code comment tags.

**Comment by krisztianb** [comment]
Requires a change in TypeDoc to be fixed: https://github.com/TypeStrong/typedoc/pull/2284

**Comment by krisztianb** [comment]
Fixed in **3.0.0**
